static/app/components/forms/controls/reactSelectWrapper.tsx

```diff
@@ -23,10 +23,7 @@ const ReactSelectWrapper = (({ref, ...props}: any) => {
 }) as any as typeof ReactSelect;
 
 export {ReactSelectWrapper as ReactSelect, ReactSelectWrapper as default, components};
-
-// biome-ignore lint/performance/noBarrelFile: not really a barrel file per say
-export {createFilter, mergeStyles} from 'react-select'; // eslint-disable-line no-restricted-imports
-// eslint-disable-next-line no-restricted-imports
```

Comment:
The `eslint-disable-next-line` might still be needed

Reply:
🫡 disabled for the whole file in 0a201e33d9e